src/js/jwt/jwt.js

```diff
@@ -294,7 +297,17 @@ function setCookieWrapper(str) {
 // Encoded WIP
 exports.getEncodedToken = () => {
     log('Getting encoded token');
-    return (priv.keycloak.token);
+    // Check if the token is expired, if expired - report to sentry
+    if (!exports.isTokenExpired()) {
+        return (priv.keycloak.token);
+    } else {
```

Comment:
What do we return if the token is expired?
Is undefined what we want?

Reply:
I'm not sure about the answer to that. I'm tempted to say to try and call a refresh and then send to a login page if that fails.